go.mod

```diff
@@ -17,20 +17,22 @@ require (
 	github.com/johnfercher/maroto v0.37.0
 	github.com/kubescape/go-logger v0.0.6
 	github.com/kubescape/k8s-interface v0.0.84
-	github.com/kubescape/opa-utils v0.0.198
+	github.com/kubescape/opa-utils v0.0.199-0.20221103115452-7f21b4b201ea
```

Comment:
`opa-utils` points to unmerged branch. Once https://github.com/kubescape/opa-utils/pull/41 is merged, I will update the version

Reply:
@amirmalka  Merged